frontend/talentsearch/src/js/components/Spinner/Spinner.stories.tsx

```diff
@@ -0,0 +1,8 @@
+import type { Meta, Story } from "@storybook/react";
+
+import Spinner from "./Spinner";
+
+export default {
+  component: Spinner,
+  title: "Spinner",
+};
```

Comment:
Cool! I think this is just the "metadata" object being exported, but you still need to export a Story in order for storybook to render one.

Something like

```tsx
const Template: Story = () => (
  <Spinner />
)

export const Default = Template.bind({});
```

Reply:
Wow 😳, thanks! Resolved with [add spinner story component export](https://github.com/GCTC-NTGC/gc-digital-talent/pull/2620/commits/f61d5c5ccd39fcb942d2b370b764cb78f32632bd)

Reply:
haha Thank YOU!